tfhe/src/integer/gpu/ciphertext/compact_list.rs

```diff
@@ -83,9 +83,9 @@ impl CudaCompactCiphertextListExpander {
         index: usize,
         streams: &CudaStreams,
     ) -> Option<(CudaRadixCiphertext, DataKind)> {
-        let preceding_infos = self.blocks_info.get(..index).unwrap();
-        let current_info = self.blocks_info.get(index).unwrap();
-        let message_modulus = self.blocks_info.get(index).unwrap().info.message_modulus;
+        let preceding_infos = self.blocks_info.get(..index)?;
```

Comment:
Why not unwrapping anymore?

Reply:
If an exception happens we want to pass it upstream. There is an assert on a test expecting for it.